scripts/validate-eks-ci-role-policy/main_test.go

```
@@ -337,6 +384,41 @@ func TestValidatePinnedUnifiSource(t *testing.T) {
+	if err := validateUnifiSigningKey(nil); err == nil {
+		t.Fatal("a missing UniFi signing key Secret must be rejected")
+	}
+	if err := validateUnifiSigningKey(append(documents, documents[0])); err == nil {
+		t.Fatal("a duplicate UniFi signing key Secret must be rejected")
+	}
+
+	tampered, err := decodeDocuments(contents)
+	if err != nil {
+		t.Fatalf("decode the tampered-key fixture: %v", err)
+	}
+	tampered[0]["stringData"].(map[string]any)["github.asc"] = "attacker key\n"
```

golangci-lint (2.12.2)

[error] type assertion must be checked

(forcetypeassert)